modules/ocf_mysql/manifests/init.pp

```diff
@@ -1,30 +1,31 @@
 class ocf_mysql {
-  file {
-    # preseed root password
-    '/opt/share/puppet/mariadb-server-10.0.preseed':
-      mode      => '0600',
-      source    => 'puppet:///private/mariadb-server-10.0.preseed',
-      show_diff => false;
 
+  include ocf::ssl::default;
+
+  user { 'mysql':
+    groups => ['ssl-cert'];
+  }
+
+  file {
     '/root/.my.cnf':
       mode      => '0600',
       source    => 'puppet:///private/root-my.cnf',
       show_diff => false;
   }
 
   class { 'ocf::packages::mysql_server':
```

Comment:
nit: it would make more sense to have this be the first thing in the file after ssl is pulled in. most things are going to have a dep on it too